tests/integration/dual-channel-replication.tcl

```diff
@@ -25,7 +25,12 @@ proc get_client_id_by_last_cmd {r cmd} {
 
 proc wait_and_resume_process idx {
     set pid [srv $idx pid]
-    wait_for_log_messages $idx {"*Process is about to stop.*"} 0 2000 1
+    wait_for_condition 50 1000 {
+        [exec ps -o state= -p $pid] eq "T" ||
+        [exec ps -o state= -p $pid] eq "Z"
```

Comment:
did you plan to catch that? I mean this is basically when we are waiting to collect the zombie, this is not the expected flow we are trying to catch...